packages/application-components/src/components/custom-views/custom-views-selector/custom-views-selector.tsx

```diff
@@ -19,18 +19,22 @@ type TCustomViewSelectorBaseProps = {
 };
 type TCustomViewSelectorProps = TCustomViewSelectorBaseProps & {
   customViewLocatorCode?: string;
+  margin?: string;
 };
 type TCustomViewSelectorWithRequiredProps = TCustomViewSelectorBaseProps & {
   customViewLocatorCode: string;
+  margin?: string;
 };
 
 type TWrapperProps = {
   shouldRender: boolean;
+  margin?: string;
```

Comment:
Nit: isn't this required?

Reply:
From the component's perspective, I don't think its needed as it would depend on the consumers needs.